core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -559,6 +559,31 @@ object TestUtils extends Logging {
     controllers.foreach(controller => ensureConsistentKRaftMetadata(brokers, controller))
   }
 
+  /**
+   * Create a topic
+   * If zkClient exists, create a topic in Zookeeper.
+   * Otherwise, create a topic using the Admin Client.
+   */
+  def createTopicWithAssignment(topic: String,
```

Comment:
Could we reuse `createTopicWithAdmin()` instead of creating a new method? It's fine to use the Admin client to create the topic even in ZK mode.

Reply:
Thank You!